index.d.ts

```diff
@@ -96,6 +107,12 @@ declare class CieManager {
    * Remove all events callbacks: onEvent / onError / onSuccess
    */
   removeAllListeners(): void;
+
+  /**
+   * only iOS.
+   * customize the info and error messages shown on NFC reading card alert
+   */
+  setAlertMessage(key: iOSAlertMessageKeys, value: string): void;
```

Comment:
What you think about change this method as:

```typescript
  setAlertMessages(ReadonlyArray<key: iOSAlertMessageKeys, value: string>): void;
```

in order to define the configuration of all the possible alert messages one time and pass in a single function instead of using multiple functions?

Not releated to this pr but in general I think could improve the usage and readability of the code define the configuration of the library and pass it in the `start` method, without using multiple methods, eg:

```typescript
type CieConfiguration = {
pin: string;
authUrl: string;
iosAlertMessage?: ReadonlyArray<key: iOSAlertMessageKeys, value: string>;
onEvent?: (event: Event) => void;
onSuccess?: (url: string) => void
onError?: (error: Error) => void
};

const cieConfig: CieConfiguration = {...};
cieManager.start(cieConfig);

```

Reply:
I'm agree, your suggestions are an improvement.
So I made a commit to address both

1. Now you can start the SDK by passing the alert messages configuration. It has effect only on iOS. I prefere a partial record instead of array, to avoid item duplication
```typescript
cieManager.start({ readingInstructions: "my test" })
```
2. I keep `setAlertMessage` so if we need to change an alert message when the sdk is already started

https://github.com/pagopa/io-cie-sdk/pull/38/commits/c39b05f0fa3ce494e38df14b037909e30efdbc18